### ISSUE TITLE
Ukrainian support: doesn't process Ї (yi) letter

### ISSUE THREAD

**Issue by daubmannus** [opendatakosovo/cyrillic-transliteration#42]
installed cyrtranslit-1.1.1

`echo 'Її' | cyrtranslit -l UA`
output:
`Ïï`
(i.e. not changed)

**Comment by petalser** [comment]
> installed cyrtranslit-1.1.1
> 
> `echo 'Її' | cyrtranslit -l UA` output: `Ïï` (i.e. not changed)

What output woul'd you expect? Is "Ji" ok?

**Comment by daubmannus** [comment]
I'd expect anything in ASCII, not cyrillic. I think 'Ji/ji' is possible option, better than keep it cyrillic.

But I have to admit also that I couldn't figure out what is the source of transliteration system here.
I's definitely not official one (https://czo.gov.ua/en/translit)

**Comment by georgeslabreche** [comment]
Hi Team! Thank you for your input. Here's a pull request with the fix:
https://github.com/opendatakosovo/cyrillic-transliteration/pull/43

@daubmannus: try it out and if it works I'll merged it in and prepare a v1.1.2 release.

I made sure it works with different casing variations:
```bash
>>> import cyrtranslit
>>> cyrtranslit.to_cyrillic("Ji", "ua")
'Ї'
>>> cyrtranslit.to_cyrillic("JI", "ua")
'Ї'
>>> cyrtranslit.to_cyrillic("ji", "ua")
'ї'
>>> cyrtranslit.to_cyrillic("jI", "ua")
'ї'
```

**Comment by georgeslabreche** [comment]
I went ahead and merged: https://github.com/opendatakosovo/cyrillic-transliteration/pull/43
